bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -55,6 +58,7 @@ public abstract class Device implements Drawable {
 	String[] loadedFonts;
 
 	volatile boolean disposed;
+	private Vector<Resource> resources = new Vector<>();
```

Comment:
```suggestion
	private Vector<Resource> resourcesWithZoomSupport = new Vector<>();
```

Also, why use `Vector` and not `Collection` / `List`? Do you need synchronization?

Reply:
Yes

Reply:
Rename?

Reply:
Done